schrift.c

```diff
@@ -65,6 +64,7 @@
 #define GOT_A_SCALE_MATRIX         0x080
 
 /* macros */
+#define ABS(x)    ((x) > 0 ? (x) : -(x))
```

Comment:
To match the semantics of the glibc implemntation this should be `((x) >= 0 ? (x) : -(x))`.

See: https://github.com/lattera/glibc/blob/master/stdlib/abs.c

As it's written, this would introduce `-0` floating point values which might cause issues?

Reply:
Interesting question. When in doubt, following the glibc implementation is probably the right choice.
So I updated the PR this way.

Reply:
Aaand there IS difference between (x < 0 ? -x : x) and  (x >= 0 ? x : -x). The former won't change the sign of a quiet NAN, the latter will :)